### DISCUSSION TITLE
New ideas or designs for stickers, flyers, website.

### DISCUSSION THREAD

**Discussion by ObjectifMoon21** [sovenor/bitcoin-rocks#41] [Ideas]
Share here new designs and ideas for new stickers and more...

**Comment by ObjectifMoon21** [comment]
let me start, in the european union there are nutri-score symbols on all supermarket food packages, that tell you what's supposed to be good for you. 

So I thought I'd make an inflation-score which shows that the euro is very bad when it comes to inflation and there's a better money you can use right now:
![1024px-Inflation-score-B](https://github.com/user-attachments/assets/21de6422-8107-4b42-9c7f-93bcbba2a220)

**Reply by ObjectifMoon21** [reply]
i've made a alternative of this sticker, not sure which one is better
![Nutri-score-B](https://github.com/user-attachments/assets/df5c9405-8640-4835-a432-fdda72e6bfc9)

**Comment by sovenor** [comment]
This is a fun idea! I prefer the latter because it follows the same "ABCDE" scale as the official NUTRI-SCORE, which makes it more of a recognizable parody. My only hesitation is that the Bitcoin "B" is not the first on the list. However I don't think that really matters if the point is to show that Bitcoin is much better than the Euro in last place.

I would also try a version where the QR code is smaller and to the right of the entire graphic. I think this will help it to more closely parody the official NUTRI-SCORE version while making the outer whitespace more consistent.

I am happy to play around with it if you can send me the source files.

Great work @ObjectifMoon21!
